.github/workflows/build-irods-http-api-ubuntu.yml

```diff
@@ -0,0 +1,54 @@
+name: iRODS HTTP API Builder
+on: [pull_request]
+jobs:
+  build:
+    name: Build iRODS HTTP API
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Install Misc
```

Comment:
Let's use the same wording as in irods/irods for this.
- `Install Prerequisites`

Reply:
Avoided that name initially since I wasn't sure what exactly are the prerequisites.

Reply:
Done.